services/workflows-service/src/workflow/workflow.service.ts

```diff
@@ -1939,6 +1943,32 @@ export class WorkflowService {
         },
         // @ts-expect-error - error from Prisma types fix
         extensions: workflowDefinition.extensions,
+        invokeRiskRulesAction: async (
+          context: object,
```

Comment:
```suggestion
          context: Record<string, unknown>,
```
`object` can be anything, including 'kaki' or 2.

Reply:
@Blokh